TreePiece.cpp

```diff
@@ -5301,13 +5301,11 @@ void TreePiece::initiatePrefetch(int chunk){
 void TreePiece::commenceCalculateGravityLocal(intptr_t d_localMoments, 
 		                              intptr_t d_localParts, 
 					      intptr_t d_localVars,
-					      intptr_t streams, int numStreams,
                                               size_t sMoments, size_t sCompactParts, size_t sVarParts) {
     if (!bUseCpu) {
       this->d_localMoments = (CudaMultipoleMoments *)d_localMoments;
```

Comment:
Since you are making a change here, you could also clean up the use of `this->`.  The `this->` should be deleted in the next 6 lines.

Reply:
See my response to the above comments.